aziotctl/aziotctl-common/src/config/super_config.rs

```diff
@@ -26,6 +26,14 @@ use url::Url;
 pub struct Config {
     pub hostname: Option<String>,
 
+    // DEVNOTE: ideally, this would be called `local_gateway_hostname`, to match
```

Comment:
I think you can `#[serde(alias)]` it so that it gets deserialized with either name, then use different names in the `template.toml`s.

Reply:
ooooh, fancy. I guess the better question is whether or not that'd be more confusing than just having a single [somewhat mismatched] name for the field?

Reply:
Should be fine. a-i-s users won't have the edge-specific file in the first place, and a-e users hopefully don't look at the a-i-s template and only at the a-e template.

Reply:
so you're thinking it makes sense to use `#[serde(alias)]` then?

Reply:
Yes. It also solves the `(?)` in

>and rephrase the "Nested Edge" part to taste (?)

because you can just describe it as the local gateway.